js/scorm/wrapper.js

```diff
@@ -301,14 +306,31 @@ define ([
 
             this.endTime = new Date();
 
+            var exit_property = "cmi.core.exit";
+            var exit_value = "";
```

Comment:
`exitState` would be more in keeping with the Adapt naming convention and the names you've used for this elsewhere e.g. `exitStateIfComplete `